apps/activejobs/app/models/jobstatusdata.rb

```diff
@@ -201,6 +203,33 @@ def extended_data_pbspro(info)
     self
   end
 
+  # Store additional data about the job. (SGE-specific)
+  #
+  # Parses the `native` info function for additional information about jobs on SGE systems.
+  #
+  # @return [Jobstatusdata] self
+  def extended_data_sge(info)
+    return unless info.native
+    attributes = []
+    attributes.push Attribute.new "Cluster", self.cluster_title
+    attributes.push Attribute.new "Job Id", self.pbsid
+    attributes.push Attribute.new "Job Name", self.jobname
+    attributes.push Attribute.new "User", self.username
+    attributes.push Attribute.new "Account", self.account
+    attributes.push Attribute.new "Partition", self.queue
+    attributes.push Attribute.new "Node List", self.nodes.join(", ") unless self.nodes.blank?
+    self.native_attribs = attributes
+
+    self.submit_args = nil
+    self.output_path = info.native[:work_dir]
+
+    output_pathname = Pathname.new(info.native[:work_dir])
+    self.file_explorer_url = build_file_explorer_url(output_pathname)
+    self.shell_url = build_shell_url(output_pathname, self.cluster)
```

Comment:
Isn't the native attribute empty?  I just glanced at SGE, so I could have overlooked it, but I can't see where (if) work directory gets set. Even if it doesn't fail and get's set to "" these URLs may be useless.